### PR TITLE
4 reviews

### PR DESCRIPTION
**Pull request by SentinelProxima**

### CODE REVIEW

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+const Dashboard = (props) => {
+    const [reviews, setReviews] = useState([]);
+    useEffect(() => {
+        fetch('https://pacific-citadel-03998.herokuapp.com/api/reviews')
+        .then(res => res.json())
+        .then((data) => {
+            console.log(data);
```
Comment (HeatherEver) [inline src/components/Dashboard.js]: please remove console logs from code when finished

File: src/components/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            setReviews(data.reviews);
+        })
+    }, []);
+    return (
+    <ul>
+        {reviews.map((item) => (
+            <div class="review-box">
+                <img class="review-image" src={item.review_img_url} alt={item.title}/>
```
Comment (HeatherEver) [inline src/components/Dashboard.js]: this should be `className="review-image"`  as `class` is used for other things within React

File: src/components/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        .then(res => res.json())
+        .then((data) => {
+            console.log(data);
+            setReviews(data.reviews);
+        })
+    }, []);
+    return (
+    <ul>
+        {reviews.map((item) => (
+            <div class="review-box">
```
Comment (HeatherEver) [inline src/components/Dashboard.js]: as we're in a `<ul>` tag, we should be rendering an `<li>` tag within the map and giving that a unique key which will get rid of the error you are seeing regarding 'each list item should have a unique key'

**Review by HeatherEver — COMMENTED** [review]
Nice work Michael! Just a couple of things to consider. One other thing to consider is how you can make your site more responsive, when I make the screen small your content currently spills off the page - is there a way you could make it more responsive? 
Good work so far ⭐

File: src/components/Dashboard.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+
+const Dashboard = (props) => {
+    const [reviews, setReviews] = useState([]);
+    useEffect(() => {
+        fetch('https://pacific-citadel-03998.herokuapp.com/api/reviews')
```
Comment (Rosem10) [inline src/components/Dashboard.js:6]: Moving forward, you might consider separating all of your fetch requests into one folder where they can all be exported from - but this works well for now!

**Review by Rosem10 — APPROVED** [review]
Looking good!

**Pull request merged**